app/script/auth/module/action/ValidationError.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export default class ValidationError extends Error {
+  constructor(params) {
+    super();
+    this.name = this.constructor.name;
+    this.label = params.label;
+    this.message = params.label;
+  }
+
+  is = label => {
```

Comment:
No `static`, no `constant`, what is `is`?

Reply:
It's a convenience method to check if an instance of an error has a given label.
`if(someValidationError.is(ValidationError.NAME.PATTERN_MISMATCH)) ...`